### ISSUE TITLE
Disallow explicit finite difference soil temperature scheme

### ISSUE THREAD

**Issue by tbohn** [UW-Hydro/VIC#37]
The explicit scheme (currently the default) is not accurate for certain combinations of time step and node spacing.  Currently VIC gives an error message if the user attempts to run the explicit scheme with these combinations.  Given that the implicit scheme (currently optional) has no such restrictions and does not require substantially more cpu time than the explicit scheme, it would seem beneficial to remove the explicit scheme.

**Comment by tbohn** [comment]
I don't understand - you really want to remove the explicit scheme for 5.0?  We still don't have a fix for frozen soil behavior; Tara Troy's code only works in the explicit mode.  I really don't think we can do this for 5.0.

**Comment by bartnijssen** [comment]
@tbohn: Joe and are sitting/standing here laughing given your comment "I don't understand ..." in an issue that only has previous comments from you. Imagine how the rest of us feel :-)

**Comment by tbohn** [comment]
Sorry about any confusion.  What exactly do you want to have happen?  Just insert logic that exits with error if explicit is selected (easy) or remove all explicit-mode code from the model (harder)?  I'm worried that this is a potential can of worms that we don't want to open before the release of 5.0.

**Comment by jhamman** [comment]
Agreed.  We can move this to 5.1.  This isn't related to the refactor and is best placed in 5.1.
